### PR TITLE
added lookout-sonarcheck-analyzer

### PR DESCRIPTION
**Pull request by rporres**
Signed-off-by: Rafael Porres Molina <rafa@sourced.tech>

### CODE REVIEW

File: lookout-sonarcheck-analyzer/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
[the remark this answers is not in the document]
Comment (rporres) [reply lookout-sonarcheck-analyzer/Chart.yaml]: So true :joy:

**Comment by rporres** [comment]
Now you've got even a README, @meyskens ;-)

File: lookout-sonarcheck-analyzer/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+```bash
+$ helm install --name my-release lookout-sonarcheck-analyzer
+```
+
+## Configuration
+
+See `values.yaml` for the available configuration parameters. Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+```sh
+$ helm install --name my-release lookout-sonarcheck-analyzer
```
Comment (meyskens) [inline lookout-sonarcheck-analyzer/README.md]: no ending ` ``` ` ?

Comment (rporres) [reply lookout-sonarcheck-analyzer/README.md]: fixed

**Pull request merged**